# Generated for NaoQi method ALSentinel.enableDefaultActionTripleClick
# enable or disable the "restart naoqi" action after three pushes.

# true of false to respectively enable or disable
bool enable
---


